eZ/Bundle/EzPublishCoreBundle/Session/Handler/NativeSessionHandler.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * File containing the SymfonyNativeSessionHandler class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Session\Handler;
+
+use Symfony\Component\HttpFoundation\Session\Storage\Handler\NativeSessionHandler as SymfonyNativeSessionHandler;
```

Comment:
Be aware this class has been deprecated in 3.x and removed in Symfony 4.

As we require PHP 5.6 on 1.13 anyway, can we just extend PHP's [SessionHandler](http://php.net/SessionHandler)?
It's what Symfony does in 2.8 conditionally, and in 3.x unconditionally: https://github.com/symfony/http-foundation/blob/2.8/Session/Storage/Handler/NativeSessionHandler.php

will that work with symfony 2? Or should we just adjust it when merging up to 7.x?

Reply:
It should work in symfony 2 too. I'll double-check

Reply:
Yeah, works. Fixed!